modules/core/src/main/scala/org/scalasteward/core/vcs/VCSRepoAlg.scala

```diff
@@ -45,10 +45,19 @@ object VCSRepoAlg {
           for {
             parent <- repoOut.parentOrRaise[F]
             _ <- gitAlg.syncFork(repo, withLogin(parent.clone_url), parent.default_branch)
+            _ <- maybeDeleteUpdateBranch(repo)
           } yield parent
         }
 
       val withLogin: Uri => Uri =
         util.uri.withUserInfo.set(config.vcsLogin)
+
+      def maybeDeleteUpdateBranch(repo: Repo): F[Unit] = {
+        val branch = Branch(updateBranchPrefix)
+        for {
+          _ <- gitAlg.deleteLocalBranch(repo, branch).attempt
+          _ <- gitAlg.deleteRemoteBranch(repo, branch).attempt
```

Comment:
Instead of `attempt`ing to delete these branches, it would be better to check if theses branches exists before deleting them and then not use `attempt` at all.